inlineplz/decorators.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+from functools import partial
+import inspect
+from inlineplz.util.system import run_command, install_linter
+from inlineplz.registry import register_linter
+
+
+class Registry(object):
+    REGISTRY = dict()
```

Comment:
i'm wondering what the naming conventions/practices are for class variables. is it normal to give them all-upper names as if they're constants?